misaligned.py

```diff
@@ -4,9 +4,12 @@ def print_color_map():
     minor_colors = ["Blue", "Orange", "Green", "Brown", "Slate"]
     for i, major in enumerate(major_colors):
         for j, minor in enumerate(minor_colors):
+            checkValidPairNumber(i * 5 + j)
             print(f'{i * 5 + j} | {major} | {minor}')
     return len(major_colors) * len(minor_colors)
-
+def checkValidPairNumber(pair_number):
+    assert(pair_number<1)

```

Comment:
These asserts will fail even after the code is fixed.
Use asserts to state the functionality of the code. The pair numbers must be 1-25, not outside. So assert that the pair numbers are within that range, not outside